Browsable FAQs/QA/flows/Browsable FAQs.md

```diff
@@ -109,14 +116,69 @@ card FetchContent, then: DisplayContent do
     )
 end
 
+card GetVariation when count(content_data.body.body.text) > 0, then: DisplayContent do
+  content_body = content_data.body.body.text.value.message
+  variations = content_data.body.body.text.value.variation_messages
+
+  # Find and apply gender variation
+  gender_variations =
+    filter(variations, &(&1.profile_field == "gender" and &1.value == contact.gender))
+
+  content_body = if(count(gender_variations) > 0, gender_variations[0].message, content_body)
+
+  # Find and apply relationship variation
+  # Relationship stored on the contact is different to the one on the CMS, so translate between them
+  relationship_mapping = [
+    ["single", "single"],
+    ["in a relationship", "in_a_relationship"],
+    ["it's complicated", "complicated"],
+    ["empty", ""],
+    ["", ""]
+  ]
+
+  relationship_status = find(relationship_mapping, &(&1[0] == contact.relationship_status))[1]
+
+  relationship_variations =
+    filter(variations, &(&1.profile_field == "relationship" and &1.value == relationship_status))
+
+  content_body =
+    if(count(relationship_variations) > 0, relationship_variations[0].message, content_body)
+
+  # Find and apply age variation
+  # We only have year of birth, so we have to make the 1 January assumption and just difference to current year
+  # Age is also stored as ranges in the CMS, so map between age and age ranges
+  year_of_birth = contact.year_of_birth or ""
+
+  age =
+    if(
+      isnumber(year_of_birth),
+      year(now()) - year_of_birth,
+      -1
+    )
+
+  age_mapping = [
+    [[15, 18], "15-18"],
+    [[19, 24], "19-24"]
+  ]
+
+  age_mapping_result = filter(age_mapping, &(age >= &1[0][0] and age <= &1[0][1]))
+  age_range = if(count(age_mapping_result) > 0, age_mapping_result[0][1], "")
+  age_variations = filter(variations, &(&1.profile_field == "age" and &1.value == age_range))
+  content_body = if(count(age_variations) > 0, age_variations[0].message, content_body)
```

Comment:
We have to choose a priority, I don't think we'll want to send both, since it's the same content, just with some personalisation. In this case I've chosen age > relationship > gender. In practice though, I haven't seen content with more than one contact field personalisation yet.